lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
Yeah, I think this change makes the whole flow a bit more stable und predictable.